### PR TITLE
Revenge-ify the tweak

### PR DESCRIPTION
**Pull request by Cristiandis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,22 @@
-# BunnyTweak
+# NOTICE!!!
```
Comment (PalmDevs) [inline README.md]: Please remove this notice

File: README.md [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
 > As of right now this tweak does not encompass some functionalities when running in a jailed environment with a wildcard certificate \
 > If you value these features sign the application with a local dev certificate:
 > - setAlternateAppIcon does not work, thus breaking dynamic app icons
-> - sharing files to the application/selecting items via the Files app does not work 
+> - sharing files to the application/selecting items via the Files app does not work
 
 ## Installation
 
-Builds can be found in the [Releases](https://github.com/pyoncord/BunnyTweak/releases/latest) tab.
+Builds can be found in the [Releases](https://github.com/Cristiandis/RevengeTweak/releases/latest) tab.
```
Comment (PalmDevs) [inline README.md]: Update this link

File: README.md [hunk 3]
```diff
@@ -58,17 +64,17 @@ bash -c "$(curl -fsSL https://raw.githubusercontent.com/theos/theos/master/bin/i
 
 If you've already installed theos, you can run `$THEOS/bin/update-theos` to make sure it's up to date.
 
-5. Clone this repository with `git clone git@github.com:pyoncord/BunnyTweak.git` and `cd` into it. Replace the URL with your fork if you've forked this repository.
+5. Clone this repository with `git clone git@github.com:Revenge/Tweak.git` and `cd` into it. Replace the URL with your fork if you've forked this repository.
```
Comment (PalmDevs) [inline README.md]: revenge-mod/RevengeTweak

File: Resources/payload-base.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
 globalThis.__PYON_LOADER__ = {
-    loaderName: "BunnyTweak",
-    loaderVersion: "0.3.1",
-    hasThemeSupport: true,
-    storedTheme: null,
-    fontPatch: 2
-}
+  loaderName: "RevengeTweak",
+  loaderVersion: "0.0.1",
```
Comment (PalmDevs) [inline Resources/payload-base.js]: Use 1.0.0 here

File: README.md (hunk 1 above)
Comment (Cristiandis) [reply README.md]: ok

File: UPDATING.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+How to update to latest upstream repo:
+
+First Time:
+
+- git remote add public https://github.com/pyoncord/BunnyTweak
```
Comment (PalmDevs) [inline UPDATING.md]: Update this link too

File: control [hunk 6]
```diff
@@ -1,11 +1,11 @@
-Package: io.github.pyoncord.app
-Name: Bunny
-Version: 0.3.1
+Package: io.github.revenge-mod.app
+Name: Revenge
+Version: 0.0.1
```
Comment (PalmDevs) [inline control:3]: 1.0.0

File: app-repo.json [hunk 7]
```diff
@@ -1,58 +1,22 @@
 {
-  "name": "Pyoncord App Repo",
-  "identifier": "io.github.pyoncord.repo",
-  "iconURL": "https://pyoncord.github.io/assets/icon-altstore.png",
+  "name": "Revenge App Repo",
+  "identifier": "io.github.revenge-mod.repo",
+  "iconURL": "https://cdn.discordapp.com/attachments/1205207690352005243/1284228073415249950/icon-altstore.png?ex=66e5de18&is=66e48c98&hm=737f59f228f7d887881fff1fcb6d8b08258b91c3c0d4df8501dfc499b51d870d&",
```
Comment (PalmDevs) [inline app-repo.json:4]: This needs to be changed. I'll have to create a branding repository. For now this can stay, I suppose.

**Review by PalmDevs — CHANGES_REQUESTED** [review]
Needs some changes.

File: app-repo.json (hunk 7 above)
Comment (Cristiandis) [reply app-repo.json:4]: once you make the branding repo ill change and pr

**Pull request merged**